modules/local/starfusion/build/main.nf

```diff
@@ -5,7 +5,7 @@ process STARFUSION_BUILD {
     conda "${moduleDir}/environment.yml"
```

Comment:
We would need to update the `environment.yaml` of conda as well. Plus the test's snapshots.